packages/plugins/jwt/src/__tests__/jwt.spec.ts

```diff
@@ -539,7 +539,7 @@ describe('jwt plugin', () => {
     const secret = 'topsecret';
     const test = createTestServer(
       {
-        singingKeyProviders: [createInlineSigningKeyProvider(secret)],
+        signingKeyProviders: [createInlineSigningKeyProvider(secret)],
```

Comment:
Can we add a test for the backward compatibility to prevent future regressions?

Reply:
You mean you want the old misspelled option should continue to work and we should have a test for this ?

Reply:
Yes, Until we completely remove it, let's make sure it works.

Reply:
Aaaah you're right ! I will fix this and add test